packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -1049,6 +1049,7 @@ describe('getInputProps', () => {
                     { label: '2', url: '#2' },
                   ],
                   source: {
+                    sourceId: 'testSource',
```

Comment:
We don't need to add a dependency to our `createSource` test util in this test.

```suggestion
                    sourceId: expect.any(String),
```